addons/event_sale/wizard/event_edit_registration.py

```diff
@@ -24,7 +27,7 @@ def default_get(self, fields):
 
         attendee_list = []
         for so_line in [l for l in sale_order.order_line if l.event_ticket_id]:
-            existing_registrations = [r for r in registrations if r.event_ticket_id == so_line.event_ticket_id]
+            existing_registrations = [r for r in registrations if r.sale_order_line_id.id == so_line.id]
```

Comment:
Mmmh ... do you know why the code uses the ticket and not the line ? Any impact, side effect, ... ?